packages/playground/src/weblets/tf_full_vm.vue

```diff
@@ -185,7 +185,11 @@ const dedicated = ref(false);
 const rentedByMe = ref(false);
 const certified = ref(false);
 const disks = ref<Disk[]>([]);
-const hasGPU = ref(false);
+const GPUToggled = ref(false);
+const hasGPU = computed(() => {
+  return (!dedicated.value && !rentedByMe.value) || GPUToggled.value;
```

Comment:
![image](https://github.com/user-attachments/assets/0f2d326a-287d-4aa2-b4ec-3e1232a0ab1f)

hasgpu should be true if the gputoggled and one of the dedicated and rented by is on